layouts/partials/demo-page-cta.html

```diff
@@ -0,0 +1,21 @@
+<div class="demo-cta__section border-b-3 demo-bg-image--right-bottom" style="background-image: url(/demo-page-cta.png);">
+    <div class="demo-cta__section-wrapper container-xl">
+        <div class="demo-cta__inner">
+            <header class="demo-cta__header mx-auto">
+                <h2 class="demo-cta__title font-unbounded">Ready to <span class="text-primary">improve</span> your <br
+                        class="d-none d-lg-block">development
+                    workflow?</h2>

```

Comment:
![Screenshot 2025-04-22 at 12 08 26](https://github.com/user-attachments/assets/e6208869-a4a9-48b2-a195-af34b19192c2)

this is slightly weird at some widths, maybe missing a space?
```suggestion
                <h2 class="demo-cta__title font-unbounded">Ready to <span class="text-primary">improve</span> your <br
                        class="d-none d-lg-block"> development
                    workflow?</h2>

```